### ISSUE TITLE
Styled components with element-ui

### ISSUE THREAD

**Issue by zuxriddinkamalov** [styled-components/vue-styled-components#90]
Can we use styled-components for styling element-ui framework components?
```
import styled from 'vue-styled-components'
import { Button } from 'element-ui'

const Button = styled(ElButton, props)`
  border: 1px solid #fff;
}
`

export default Button
```
![image](https://user-images.githubusercontent.com/28143150/67164825-fce52e00-f397-11e9-978c-8d6a063e231e.png)
How to solve?

**Comment by codebender828** [comment]
No, you cannot. Components imported from an external library that were not natively created with `vue-styled-components` cannot be "styled" with `styled-components`.

**Comment by JessicaSachs** [comment]
@codebender828 This used to be possible. I'm running vue-styled-components@1.4.5 and I extend element-ui components inside of it.

I will caution people who want to do this -- it's not easy or extensible to override element-ui's (or any other framework's) styles. The specificity of their styles changes and can break your UI.

**Comment by liqueflies** [comment]
Fixed in #96.
